SignalServiceKit/src/Storage/TSDatabaseView.m

```diff
@@ -295,30 +300,26 @@ + (YapDatabaseViewSorting *)messagesSorting {
 
 + (void)asyncRegisterSecondaryDevicesDatabaseView:(OWSStorage *)storage
 {
-    YapDatabaseViewGrouping *viewGrouping =
-        [YapDatabaseViewGrouping withObjectBlock:^NSString *_Nullable(YapDatabaseReadTransaction *_Nonnull transaction,
-            NSString *_Nonnull collection,
-            NSString *_Nonnull key,
-            id _Nonnull object) {
-            if ([object isKindOfClass:[OWSDevice class]]) {
-                OWSDevice *device = (OWSDevice *)object;
-                if (![device isPrimaryDevice]) {
-                    return TSSecondaryDevicesGroup;
-                }
+    YapDatabaseViewGrouping *viewGrouping = [YapDatabaseViewGrouping withObjectBlock:^NSString *_Nullable(
+        YapDatabaseReadTransaction *transaction, NSString *collection, NSString *key, id object) {
+        if ([object isKindOfClass:[OWSDevice class]]) {
+            OWSDevice *device = (OWSDevice *)object;
+            if (![device isPrimaryDevice]) {
+                return TSSecondaryDevicesGroup;
             }
-            return nil;
-        }];
+        }
+        return nil;
+    }];
 
     YapDatabaseViewSorting *viewSorting =
-        [YapDatabaseViewSorting withObjectBlock:^NSComparisonResult(YapDatabaseReadTransaction *_Nonnull transaction,
-            NSString *_Nonnull group,
-            NSString *_Nonnull collection1,
-            NSString *_Nonnull key1,
-            id _Nonnull object1,
-            NSString *_Nonnull collection2,
-            NSString *_Nonnull key2,
-            id _Nonnull object2) {
-
+        [YapDatabaseViewSorting withObjectBlock:^NSComparisonResult(YapDatabaseReadTransaction *transaction,
+            NSString *group,
+            NSString *collection1,
+            NSString *key1,
+            id object1,
+            NSString *collection2,
+            NSString *key2,
+            id object2) {
```

Comment:
I'm assuming this is clang-format gone wild.

Reply:
DatabaseView.m

Reply:
Seems that way.